### DISCUSSION TITLE
T-POT Wazuh integration

### DISCUSSION THREAD

**Discussion by Cyb3rD0c** [telekom-security/tpotce#1631] [General]
Hi,
    I wanted to ask you two questions. I am conducting a research and utilising t-pot for that, is there an already available integration of t-pot with wazuh? secondly wanted to ask you how the vulnerability data shown in t-pot what's the mechanism behind it? can I access its documentation for understanding? and where are the logs consolidated in tpot? what I have noticed is, logs at /var/lib/docker/conatiners/*/*-json.log gets deleted on restarts so where are the actual logs stored, if I want to retrieve them. I would appreciate an early reply.

**Comment by dmille6** [comment]
you would need to write some kind of script to pull the info you wanted from the hive. (there are a few discussions about using Curl and python to do this) then use the wazzah API to push that data into wazzah.

**Reply by dmille6** [reply]
honestly.. i typed "wazah example of sending an alert to wazah" in chatGPT and it gave some pretty straight forward examples. i don't think it would be hard to do.
